GoogleSheetI18n.Api.SimpleWebApi/ClientApp/package.json

```diff
@@ -39,7 +43,8 @@
     "build": "react-scripts build",
     "test": "cross-env CI=true react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "lint": "eslint ./src/"
+    "lint": "eslint ./src/",
+    "babel": "./node_modules/.bin/babel --extensions .babelrc src/**/*.{js,jsx,ts,tsx} --source-maps inline"
```

Comment:
Could you rename it to something more meaningful.

Reply:
suggest something